sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -1,144 +1,18 @@
-CREATE OR REPLACE TABLE
-  fxa_users_services_first_seen_v1
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_users_services_first_seen_v1`
 PARTITION BY
-  DATE(first_service_timestamp)
+  first_seen_date
 CLUSTER BY
   service,
-  user_id
+  user_id,
+  os_name
```

Comment:
order matters for clustering fields, and because user_id is unique per row, adding any fields after it should have no useful impact

Reply:
it's not unique, but there should be very few entries per user. Thanks for pointing this out, I will remove it now.